daffodil-sapi/src/test/scala/org/apache/daffodil/example/TestScalaAPI.scala

```diff
@@ -889,7 +891,7 @@ class TestScalaAPI {
     val inputSAX = new InputSourceDataInputStream(fisSAX)
     val inputDP = new InputSourceDataInputStream(fisDP)
     val bosDP = new ByteArrayOutputStream()
-    val outputter = new XMLTextInfosetOutputter(bosDP, pretty = true)
+    val outputter = new XMLTextInfosetOutputter(bosDP, true, XMLOutputStyle.Standard)
```

Comment:
Why even pass XMLOutputStyle.Standard when it's the default?  I also like pretty = true better (self-documenting) than true by itself.